tools/transformer/src/transform.js

```diff
@@ -19,9 +19,9 @@ doc = applyDiscriminatorTransformations(doc);
 doc = applyOneOfTransformations(doc);
 doc = applyAllOfTransformations(doc);
 
-doc = applyModelNameTransformations(doc, "ApiAtlas", "View", ignoredModelNames);
-doc = applyModelNameTransformations(doc, "Api", "View", ignoredModelNames);
-doc = applyModelNameTransformations(doc, "", "View", ignoredModelNames);
 doc = applyModelNameTransformations(doc, "Api", "", ignoredModelNames);
+doc = applyModelNameTransformations(doc, "Atlas", "", ignoredModelNames);
```

Comment:
Core of the change. Basically apart from the Manual views and ApiUsers there are no extra objects renamed.